pkg/vm/interop.go

```diff
@@ -50,3 +50,127 @@ func RuntimeDeserialize(vm *VM) error {
 
 	return nil
 }
+
+// EnumeratorCreate handles syscall Runtime.Enumerator.Create.
+func EnumeratorCreate(v *VM) error {
+	data := v.Estack().Pop().Array()
+	v.Estack().Push(&Element{
+		value: NewInteropItem(&arrayWrapper{
+			index: -1,
```

Comment:
why do we start with such index?

Reply:
Because of the logic in `Next`. As in C# implementation, we must call `Next` at least once before using the iterator.